engine/tests/phpunit/unit/Elgg/Notifications/NotificationsServiceUnitTestCase.php

```diff
@@ -333,6 +328,7 @@ public function testProcessQueueThreeEvents() {
 		$this->session->removeLoggedInUser();
 
 		$this->assertEquals(3, $this->notifications->processQueue($this->time + 10));
+		_elgg_services()->reset('subscriptions');
```

Comment:
if a test fails the reset isn't executed and you leave behind garbage

Reply:
not going to fix... a lot more is broken in these tests if we move it to up/down functions